packages/sdk/src/ClientSide.ts

```diff
@@ -13,7 +13,7 @@ export class ClientSide {
 	/**
 	 * @see https://docs.voucherify.io/reference/#vouchers-validate
 	 */
-	public validate(params: T.ClientSideValidateParams) {
+	public validate(params: T.ClientSideValidateParams | string) {
```

Comment:
Add trimming of `code` pleas so ` MY_CODE ` will be trimmed to `MY_CODE`.
Add removal of new line characters, feed return and similar character via:
`code.replace(/[\r\n\t\f\v]/g, "");`
Similar code is in original SDK:
https://github.com/rspective/voucherify.js/blob/master/lib/voucherify.js#L225